tictactoe.py

```diff
@@ -1,26 +1,63 @@
+empty_squares = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+playerX_moves = []
+player0_moves = []
+winningcombo_1 = [1, 2, 3]
```

Comment:
Maybe it would make sense to collect all these lists into a list? That'd be a lot less typing. If you look at the variable names here, they look pretty similar to `winningcombos[1]`, `winningcombos[2]` etc.

Reply:
🙈 I really tried doing exactly that last week, but couldn’t get it working and just resorted to this.
(Today lists in list was a piece of cake, which is kind of fascinating!)